torch/linalg/__init__.py

```diff
@@ -400,6 +400,53 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+solve = _add_docstr(_linalg.linalg_solve, r"""
+linalg.solve(input, other, *, out=None) -> Tensor
+
+Computes the solution ``x`` to the matrix equation ``matmul(input, x) = other``
+with a square matrix, or batches of such matrices, :attr:`input` and multiple right-hand side vectors :attr:`other`.
+If :attr:`input` is batched and :attr:`other` is not, then :attr:`other` is broadcasted
+to have the same batch dimensions as :attr:`input`.
+The resulting tensor has the same shape as broadcasted :attr:`other`.
+
+Supports input of ``float``, ``double``, ``cfloat`` and ``cdouble`` dtypes.
+
+.. note:: If :attr:`input` is a non-invertible matrix or non-square matrix then a RuntimeError will be thrown.
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+
```

Comment:
The NumPy documentation has a callout to lstsq:

https://numpy.org/doc/stable/reference/generated/numpy.linalg.solve.html?highlight=solve#numpy.linalg.solve

I guess we can't do this, however, because torch.lstsq != np.linalg.lstsq? When we add torch.linalg.lstsq we should remember to update this doc, however.